src/compute_horde_facilitator_sdk/sdk.py

```diff
@@ -0,0 +1,103 @@
+from typing import Any
```

Comment:
please use apiver i.e.

move this module to `_internal/` or rename it to `_sdk`
and expose public elements of it in `v1` module